op-deployer/pkg/deployer/bootstrap/implementations.go

```diff
@@ -154,7 +155,32 @@ func ImplementationsCLI(cliCtx *cli.Context) error {
 	if err := jsonutil.WriteJSON(dio, ioutil.ToStdOutOrFileOrNoop(outfile, 0o755)); err != nil {
 		return fmt.Errorf("failed to write output: %w", err)
 	}
-	return nil
+
+	if !cliCtx.Bool(deployer.AutoVerifyFlag.Name) {
+		return nil
+	}
+
+	if outfile == "" {
+		return nil
+	}
```

Comment:
Same comment as with "bootstrap superchain". We should still be able to verify contracts even if the outfile is not provided